cluster/gce/util.sh

```diff
@@ -872,7 +872,7 @@ function construct-windows-kubelet-flags {
   # double-quotes, because they still break sc.exe after expansion in the
   # binPath parameter, and single-quotes get parsed as characters instead of
   # string delimiters.
```

Comment:
This comment no longer applies directly to this flag. Perhaps move it up above and leave it as a general cautionary note in case we add empty flags in the future?

Reply:
Updated.